src/utils/api.js

```diff
@@ -0,0 +1,52 @@
+import fetch from 'isomorphic-fetch';
+
+const API_PREFIX = "/api/v0";
+
+export default class OutriggerHttp {
+    constructor(outriggerUrl) {
+        if (!outriggerUrl.startsWith('http')) {
+            throw new Error('Url scheme must be \'http\' or \'https\'');
+        }
+
+        this.outriggerUrl = outriggerUrl;
```

Comment:
How about `this.url = ${this.outriggerUrl}${API_PREFIX}/` and then you can just use `fetch(${this.url}prepare)` etc. in the methods below.